functions.js

```diff
@@ -1,30 +1,51 @@
 const superheros = [
-    {
-        name: 'IronMan',
-        date: '03/01/1963'
-    },
-    {
-        name: 'SpiderMan',
-        date: '03/07/1961'
-    },
-    {
-        name: 'Captain America',
-        date: '03/08/1941'
-    },
-    {
-        name: 'Doctor Strange',
-        date: '07/20/1963'
-    },
-    {
-        name: 'Shang-Chi',
-        date: '12/05/1973'
-    }
-]
+  {
+    name: "IronMan",
+    date: "03/01/1963",
+  },
+  {
+    name: "SpiderMan",
+    date: "03/07/1961",
+  },
+  {
+    name: "Captain America",
+    date: "03/08/1941",
+  },
+  {
+    name: "Doctor Strange",
+    date: "07/20/1963",
+  },
+  {
+    name: "Shang-Chi",
+    date: "12/05/1973",
+  },
+];
 
 function getSortedSuperheros(heros) {
-    // your code goes here
+  // your code goes here
+  heros.sort((a, b) => {
+    const firstDate = new Date(a.date).valueOf();
+    const secondDate = new Date(b.date).valueOf();
+
+    return firstDate - secondDate;
+  });
+
+  return heros;
```

Comment:
I was having problems with this exercise because I was not returning heros argument